docs/team/zhongming.md

```diff
@@ -0,0 +1,27 @@
+## Zhong Ming - Project Portfolio Page
+
+## Overview
+ietBook is a Command Line Interface (CLI) desktop application designed mainly _NUS students staying on campus_. It helps users **track their food and nutritional intake** as well as provide them with their **daily calorie recommendation**. It also has a **database prepopulated with food items commonly found around NUS** so that thse food items can be easily added to the list of food items consumed for tracking. DietBook is written mainly in Java.
+
+### Summary of Contributions
+
+#### Code contributed
+
+* [RepoSense Link](https://nus-cs2113-ay2021s1.github.io/tp-dashboard/#breakdown=true&search=snowbanana12345&sort=groupTitle&sortWithin=title&since=2020-09-27&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other)
+* Back end coding [PRs](https://github.com/AY2021S1-CS2113-T14-4/tp/pulls?q=is%3Apr+author%3Asnowbanana12345)
+
+#### Features implemented
+
+* Implemented [`dataBase package`](https://github.com/AY2021S1-CS2113-T14-4/tp/tree/master/src/main/java/seedu/dietbook/database)
+and [`database resource`](https://github.com/AY2021S1-CS2113-T14-4/tp/tree/master/src/main/resources)
+* Implemented['saveload package'](https://github.com/AY2021S1-CS2113-T14-4/tp/tree/master/src/main/java/seedu/dietbook/saveload)
+
+#### Documentation
+
+**Developer Guide**
```

Comment:
Perhaps include links to some PRs which has codes related to this.

Reply:
Perhaps you would like to include which portions of the UG you worked on as well.